backend/internal/controller/user_controller.go

```diff
@@ -49,6 +49,9 @@ func NewUserController(group *gin.RouterGroup, authMiddleware *middleware.AuthMi
 
 	group.DELETE("/users/:id/profile-picture", authMiddleware.Add(), uc.resetUserProfilePictureHandler)
 	group.DELETE("/users/me/profile-picture", authMiddleware.WithAdminNotRequired().Add(), uc.resetCurrentUserProfilePictureHandler)
+
+	group.PUT("/users/:id/disable", authMiddleware.Add(), uc.disableUserHandler)
```

Comment:
Should there be a single route such as `/users/:id/set` which allows passing options in the body, for example `{“enabled”:false}`? In the future there may be other options passed to users